src/main/java/com/google/devtools/build/lib/bazel/BazelStartupOptionsModule.java

```diff
@@ -33,10 +33,19 @@
         valueHelp = "<path>",
         help =
             "The location of the user .bazelrc file containing default values of "
-                + "Bazel options. If unspecified, Bazel uses the first .bazelrc file it finds in "
+                + "Bazel options. "
+                + "/dev/null indicates that all further `--bazelrc`s will be ignored, "
+                + "This is useful to disable the search for a user rc file, "
```

Comment:
"This" should be lower case since it follows a comma.

Reply:
replaced with "which" otherwise it has to be ", and this".